python/python/glide/async_commands/core.py

```diff
@@ -2621,6 +2621,34 @@ async def zmscore(
             await self._execute_command(RequestType.ZMScore, [key] + members),
         )
 
+    async def zdiffstore(self, destination: str, keys: List[str]) -> int:
+        """
+        Calculates the difference between the first sorted set and all the successive sorted sets at `keys` and stores
+        the difference as a sorted set to `destination`, overwriting it if it already exists. Non-existent keys are
+        treated as empty sets.
+
+        When in Cluster mode, all keys must map to the same hash slot.
```

Comment:
should we add all keys and destination? o is it clear?

Reply:
I've added `destination` as well, just in case